markdown/resume.md

```diff
@@ -0,0 +1,74 @@
+<div class="header">
+  <img src="images/logo.png" alt="logo" style="float: right; max-height: 100px;padding-right: 70px;padding-left:70px;"/>
+  <h1 src="position: relative; top: 18px;left: 10px;">Student Name</h1>
+</div>
+
+<div id="webaddress">
+<a href="mailto:emailid@domain.org">emailid@domain.org</a>
+| <a href="linkedin.com">LinkedIn</a>
+</div>
+
+## Statement of Purpose
+
+Clearly state the statement of purpose. For example, "I am a machine learning enthusiast who is always eager to learn new stuff. Seeking internship opportunities in Machine Learning while."
+
+## Education
+
+|Program|Institution  |%/CGPA|Year of completion|
+|-------|-------------|------|------------------|
+|B Tech |IIT, Palakkad|8.50  |2020 |
+|XII	|Board name	  |90%   |2016 |
+|X		|Board name   |92%   |2014 |
+
+## Scholastic Achievements
+
+* ___(2012)___ Best science project at National Science Fair.
+* ___(Year)___ Received a particular honour.
+
+## Technical Skills
+
+* ___Advanced___ __:__ C, Java, Python, MATLAB, LATEX
+* ___Beginner___ __:__ C++, Haskell, HTML, CSS
+
+## Projects
+
+* __Project 1:__ _(Jan, 2018 - Feb, 2018)_ Designed and implemented a Hashing protocol in Haskell for encrypting and decrypting messages on LAN. Find the implementation [here](github.com).
```

Comment:
I can see that this is one of your projects. I would prefer a fictitious project (some joke project is better).